torchtune/models/clip/_component_builders.py

```diff
@@ -181,6 +179,61 @@ def clip_vision_encoder(
     )
 
 
+def clip_text_encoder(
+    vocab_size: int = 49408,
```

Comment:
we don't normally keep defaults in the component builders right? since these values correspond to the specific ViT large patch